wikipedia-sentences-query-while-indexing/README.md

```diff
@@ -0,0 +1,152 @@
+
+Alternatively, you can query the REST API with whatever client you are comfortable with, `cURL`, `Postman` etc.
+The query format is as follows:
+
+```sh
+curl -X POST -d '{"data": [{"text":"hello world"}]}' http://0.0.0.0:9001/search
+```
+
+Optionally, if you have [`jq`](https://stedolan.github.io/jq/), you can just get the text of the matches of the query:
+
+```sh
+curl -X POST -d '{"data": [{"text":"hello world"}]}' http://0.0.0.0:9001/search | jq -r '.search.docs[] | .matches[] | .text'
+```
+
+## 🔮 Overview of the files
+
+*Add a list with all folders/files in the example:*
+
+| File or folder |  Contents |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| 📂 `data/`      | Folder where the data files are stored   |
+| 📂 `flows/`          | Folder to store Flow configuration                                                                               |
+| --- 📃 `dbms.yml`     | YAML file to configure DBMS (Index) Flow                                                                             |
+| --- 📃 `query.yml`     | YAML file to configure Querying Flow                                                                             |
+| 📂 `pods/`           | Folder to store Pod configuration                                                                                |
+| --- 📃 `dbms_indexer.yml`   | YAML file to configure the DBMS Indexer                                                                               |
+| --- 📃 `encoder.yml`   | YAML file to configure encoder Pod                                                                               |
+| --- 📃 `query_indexer.yml`   | YAML file to configure the Query Indexer                                                                               |
+| 🐍 `app.py`      | Code file for the example   |
+
+
+_________
+
+## ⏭️ Next steps
+
+Did you like this example and are you interested in building your own? For a detailed tutorial on how to build your Jina app check out [How to Build Your First Jina App](https://docs.jina.ai/chapters/my_first_jina_app/#how-to-build-your-first-jina-app) guide in our documentation. 
+
+If you have any issues following this guide, you can always get support from our [Slack community](https://join.slack.com/t/jina-ai/shared_invite/zt-dkl7x8p0-rVCv~3Fdc3~Dpwx7T7XG8w) .
+
+## 👩‍👩‍👧‍👦 Community
+
+- [Slack channel](slack.jina.ai) - a communication platform for developers to discuss Jina.
+- [LinkedIn](https://www.linkedin.com/company/jinaai/) - get to know Jina AI as a company and find job opportunities.
+- [![Twitter Follow](https://img.shields.io/twitter/follow/JinaAI_?label=Follow%20%40JinaAI_&style=social)](https://twitter.com/JinaAI_) - follow us and interact with us using hashtag `#JinaSearch`.  
+- [Company](https://jina.ai) - know more about our company. We are fully committed to open-source!
+
+## 🦄 License
```

Comment:
Should we add a troubleshooting section containing the two issues I found? 
1. When running JinaD from a virtual environment. Make sure the directory path in your terminal matches the JinaD directory path printed.  
2. On some Mac machines, you may have to run JinaD with sudo command.